### DISCUSSION TITLE
radicale behind lighttpd reverse proxy

### DISCUSSION THREAD

**Discussion by ambipur76** [Kozea/Radicale#1437] [Q&A reverse proxy]
Hello,

     has anybody tried to use lighttpd web server as reverse proxy for radicale? The documentation deals only with apache and nginx. Thanks in advance!

**Comment by ikwyl6** [comment]
yes. You create a "host" directive and then a proxy.server directive within that "host" block. Something like this:

$HTTP["url"] =~ "/cal" {
proxy.server = ("" => ((
    "host" => "localhost",
        "port" => 5232
        )))
}

**Comment by bobrippling** [comment]
Yes - see #1161

**Comment by ikwyl6** [comment]
The only thing I thought didn't work was links within radicale didn't work with the reverse proxy address when it was a remote (non localhost) address. I didn't go any further as I couldn't find a work around..
